README.md

```diff
@@ -90,14 +90,13 @@ const CSS = await collectCSS();
 const isEnabled = isDarkReaderEnabled();
 ```
 
-Be aware, that darkreader will add the `chrome` object onto the `window` object, these are to stub certain functions that
-the code will use. They originate from the webextension-api.
+Be aware that darkreader will add the chrome object onto the window object. These are to stub certain functions that the code will use. They originate from the webextension-api.
```

Comment:
I think we should keep `

Reply:
You are 100% correct. I stupidly removed them without even noticing. Completely my mistake.